sklearn/cluster/tests/test_affinity_propagation.py

```diff
@@ -239,10 +247,10 @@ def test_affinity_propagation_random_state():
 
 
 @pytest.mark.parametrize("centers", [csr_matrix(np.zeros((1, 10))), np.zeros((1, 10))])
-def test_affinity_propagation_convergence_warning_dense_sparse(centers):
+def test_affinity_propagation_convergence_warning_dense_sparse(centers, global_dtype):
     """Non-regression, see #13334"""
```

Comment:
I think that we should have a better description here

```
"""
Check that having sparse or dense `centers` format should not
influence the convergence.
Non-regression test for gh-13334.
"""
```

Reply:
I agree, but this is not related to this PR goal.

Reply:
A single PR for this is an overkilled IMO.